examples/with-passwordless/api-server.js

```diff
@@ -0,0 +1,559 @@
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td valign="middle" style="text-align: center; padding-top: 20px; padding-bottom: 35px; font-family: 'Rubik', sans-serif; font-weight: 300; font-size: 17px; line-height: 20px; color: #626262;">
+                                        <p style="margin: 0;">This email was meant for ${email}</p>
+                                    </td>
+                                </tr>
+    
+                            </table>
+                        </td>
+                    </tr>
+                    <!-- 1 Column Text + Button : END -->
+    
+                    <!-- Clear Spacer : BEGIN -->
+                    <!-- <tr class="browser-only" style="padding:20px;">
+                        <td aria-hidden="true" height="40" style="font-size: 0px; line-height: 0px;">
+                            &nbsp;
+                        </td>
+                    </tr> -->
+                    <!-- Clear Spacer : END -->
+                </table>
+                <!-- Email Body : END -->
+            <!--[if mso | IE]>
+            </td>
+            </tr>
+            </table>
+            <![endif]-->
+            </center>
+        </body>
+        </html>
+    `;
+}
+
+const APP_NAME = "SuperTokens Demo App"; // TODO: Your app name
+supertokens.init({
+    framework: "express",
+    supertokens: {
+        // TODO: This is a core hosted for demo purposes. You can use this, but make sure to change it to your core instance URI eventually.
+        connectionURI: "https://try.supertokens.com",
+        apiKey: "<REQUIRED FOR MANAGED SERVICE, ELSE YOU CAN REMOVE THIS FIELD>",
+    },
+    appInfo: {
+        appName: APP_NAME, // TODO: Your app name
+        apiDomain, // TODO: Change to your app's API domain
+        websiteDomain, // TODO: Change to your app's website domain
+    },
+    recipeList: [
+        Passwordless.init({
+            contactMethod: "EMAIL_OR_PHONE",
+            flowType: "USER_INPUT_CODE_AND_MAGIC_LINK",
+            createAndSendCustomEmail: async function (input) {
+                let htmlBody = getEmailBody(
+                    APP_NAME,
+                    Math.ceil(input.codeLifetime / 1000),
+                    input.urlWithLinkCode,
+                    input.userInputCode,
+                    input.email
+                );
+                await mailTransporter.sendMail({
+                    html: htmlBody,
+                    to: input.email,
+                    from: `Team Supertokens <${process.env.NODEMAILER_USER}>`,
+                    sender: process.env.NODEMAILER_USER,
+                    subject: `Login to ${APP_NAME}`,
+                });
+            },
+            createAndSendCustomTextMessage: async function (input) {
+                /*
```

Comment:
Add a comment explaining what the below commented code is and why it's commented